[2021-04-25 09:56:00,415] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: recent_played_ingestion.delete_cluster_dataproc 2021-04-25T08:00:00+00:00 [queued]>
[2021-04-25 09:56:00,433] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: recent_played_ingestion.delete_cluster_dataproc 2021-04-25T08:00:00+00:00 [queued]>
[2021-04-25 09:56:00,434] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-04-25 09:56:00,435] {taskinstance.py:1043} INFO - Starting attempt 2 of 2
[2021-04-25 09:56:00,435] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-04-25 09:56:00,444] {taskinstance.py:1063} INFO - Executing <Task(PatchedDataprocDeleteClusterOperator): delete_cluster_dataproc> on 2021-04-25T08:00:00+00:00
[2021-04-25 09:56:00,448] {standard_task_runner.py:52} INFO - Started process 135 to run task
[2021-04-25 09:56:00,452] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'recent_played_ingestion', 'delete_cluster_dataproc', '2021-04-25T08:00:00+00:00', '--job-id', '59', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/recent_played_ingestion.py', '--cfg-path', '/tmp/tmp36_2scto', '--error-file', '/tmp/tmpr0qfjo77']
[2021-04-25 09:56:00,455] {standard_task_runner.py:77} INFO - Job 59: Subtask delete_cluster_dataproc
[2021-04-25 09:56:00,505] {logging_mixin.py:104} INFO - Running <TaskInstance: recent_played_ingestion.delete_cluster_dataproc 2021-04-25T08:00:00+00:00 [running]> on host 67976f37379c
[2021-04-25 09:56:00,550] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=recent_played_ingestion
AIRFLOW_CTX_TASK_ID=delete_cluster_dataproc
AIRFLOW_CTX_EXECUTION_DATE=2021-04-25T08:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-25T08:00:00+00:00
[2021-04-25 09:56:00,561] {dataproc.py:840} INFO - Deleting cluster: ephemeral-recent-played-spark-cluster-20210425
[2021-04-25 09:56:28,665] {dataproc.py:852} INFO - Cluster deleted.
[2021-04-25 09:56:28,678] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=recent_played_ingestion, task_id=delete_cluster_dataproc, execution_date=20210425T080000, start_date=20210425T095600, end_date=20210425T095628
[2021-04-25 09:56:28,714] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-04-25 09:56:28,739] {local_task_job.py:146} INFO - Task exited with return code 0
